codalab/model/bundle_model.py

```diff
@@ -2910,18 +2961,71 @@ def create_bundle_store(self, user, name, storage_type, storage_format, url, aut
             connection.execute(cl_bundle_store.insert().values(bundle_store_value))
         return uuid
 
-    def save_oauth2_auth_code(self, grant):
-        with self.engine.begin() as connection:
-            result = connection.execute(oauth2_auth_code.insert().values(grant.columns))
-            grant.id = result.lastrowid
-        return grant
-
-    def delete_oauth2_auth_code(self, auth_code_id):
+    def update_bundle_store(self, user, uuid, update_fields):
+        """
+        update a bundle store
+        """
         with self.engine.begin() as connection:
             connection.execute(
-                oauth2_auth_code.delete().where(oauth2_auth_code.c.id == auth_code_id)
+                cl_bundle_store.update()
+                .where(and_(cl_bundle_store.c.uuid == uuid, cl_bundle_store.c.owner_id == user,))
```

Comment:
What is `user`? Should we rename it to `user_id`? Can you also rename other parameters called `user` appropriately in other methods as well?